compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -441,6 +450,30 @@ static LogicalResult setRootConfigForPackPeel4LevelTilingPipeline(
     packedSizesL0[nDims.back()] = packPeelTiling.n0Pack;
     packedSizesL0[kDims.back()] = packPeelTiling.k0Pack;
 
+    // TODO(avarma): This is currently a workaround for 1x1 AIE array to make
+    // those 2D matmul shapes work for which all of the operands get pulled in
+    // to L2 buffer. Once reprogramming of DMA ops is supported, we can get rid
```

Comment:
I didn't follow what's the problem here. What's the error message?

Reply:
Without this we get `'aie.memtile_dma' op could not find and assign a valid BD id` - because of exploding `memtile_dma` issue, which should be solved by having DMA ops reconfigured. But since the support is not yet available, this is a temporary workaround.